src/components/AccessControlNew.tsx

```diff
@@ -73,12 +80,14 @@ const AccessControlNew = () => {
         return {
             ID: formRule.ID,
             Name: formRule.Name,
+            Description: formRule.Description,
             Source,
             Destination,
             sourcesNoId,
             destinationsNoId,
             groupsToSave,
-            Flow: formRule.Flow
+            Flow: formRule.Flow,
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `······`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      Flow: formRule.Flow,
```